templates/_icons.css

```diff
@@ -1,11 +1,11 @@
 @font-face {
 	font-family: "<%= fontName %>";
-	src: url('<%= fontPath %><%= fontName %>.eot');
-	src: url('<%= fontPath %><%= fontName %>.eot?#iefix') format('eot'),
-		url('<%= fontPath %><%= fontName %>.woff2') format('woff2'),
-		url('<%= fontPath %><%= fontName %>.woff') format('woff'),
-		url('<%= fontPath %><%= fontName %>.ttf') format('truetype'),
-		url('<%= fontPath %><%= fontName %>.svg#<%= fontName %>') format('svg');
+	src: url('<%= fontPath %><%= fontName %>.eot<%= cacheBusterQueryString %>');
+	src: url('<%= fontPath %><%= fontName %>.eot?<%= cacheBuster %>#iefix') format('eot'),
```

Comment:
This is because for EOT format, the IE fix needs to always have the `?` of the querystring whether or not a cache buster is used. When it is not, for other formats the `?` is ommitted but not for EOT.

This is the only difference between the two vaiables `cacheBusterQueryString` and `cacheBuster`: the former include the `?`, the latter doesn't. I used to variables for this because having `if`s in the template made the whole thing less readable (too much clutter).

(Sorry for the late response!)